src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -20,20 +25,36 @@ public class LogUtils {
     public static final Path LOG_FOLDER = Paths.get("logs");
     public static final String START_TIME = LocalDateTime.now().format(FILENAME_DATE);
 
-    public static void setOutputToFile() {
+    public static void setOutputToFileAndConsole() {
         if (!Files.exists(LOG_FOLDER)) createFolder();
         else removeOld();
 
         try {
-            PrintStream output = getLogger();
-            System.setOut(output);
-            System.setErr(output);
-        } catch (FileNotFoundException | UnsupportedEncodingException e) {
-            System.out.println("Failed to redirect logs, file not found:");
+            OutputStream fileLogger = new FileOutputStream("logs/" + START_TIME + ".log");
+
+            System.setOut(createPrintStream(new FileOutputStream(FileDescriptor.out), fileLogger));
+            System.setErr(createPrintStream(new FileOutputStream(FileDescriptor.err), fileLogger));
+        } catch (FileNotFoundException e) {
+            System.out.println("Failed to redirect logs, file not found: " + "logs/" + START_TIME + ".log");
```

Comment:
File not found would be for the parent folder, not the log. Regardless, you shouldn't assume the exception source, just either use e.getMessage, or let the following printStackTrace explain the *actusl* error